phobos/README.md

```diff
@@ -0,0 +1,110 @@
+# Install rpc-ceph in the Phobos lab OnMetal 
+
+## Get Access
+
+Request VPN access and a cloud login to the phobos lab from [Jay Payne](mailto:jay.payne@rackspace.com)
```

Comment:
No problem I can add this information.  Along with contacting Shannon or Keith.  Is there a mail alias or a slack channel to send the request instead of an individual?

Reply:
the slack channel `#rpc-ceph` is probably good to mention